lambdas/service/app.py

```diff
@@ -125,7 +121,7 @@
         # changes and reset the minor version to zero. Otherwise, increment only
         # the minor version for backwards compatible changes. A backwards
         # compatible change is one that does not require updates to clients.
-        'version': '14.2',
+        'version': '15.0',
```

Comment:
```suggestion
        'version': '14.3',
```

External clients are not supposed to use.